roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1303,7 +1304,7 @@ def get_defaults(self, roles):
 
         common = dict(use_openshift_sdn=True, ip=ip_addr, public_ip=ip_addr,
                       deployment_type='origin', hostname=hostname,
-                      public_hostname=hostname)
+                      public_hostname=hostname, portal_net='172.30.0.0/16')
```

Comment:
With this pr it's `openshift_portal_net` prior to this pr it's `openshift_master_portal_net` assuming people are ok with that change.